reference/constraints/Bic.rst

```diff
@@ -131,4 +133,20 @@ For example, if you want to compare the ``$bic`` property of some object
 with regard to the ``$iban`` property of the same object, use
 ``propertyPath="iban"`` in the comparison constraint of ``$bic``.
 
+allowLowerCase
+~~~~~~~~~~~~~~~~
```

Comment:
The title underline should match the length of the title. Can you please remove characters to make sure they have the same length? (same applies to the title below)